Looking at the reports there's lots of bean wiring issues that stem back to:
```
Caused by: liquibase.exception.ChangeLogParseException: classpath:/db/changelog/db.changelog-master.yaml does not exist
	at liquibase.parser.core.yaml.YamlChangeLogParser.parse(YamlChangeLogParser.java:36)
	at liquibase.command.core.helpers.DatabaseChangelogCommandStep.lambda$getDatabaseChangeLog$0(DatabaseChangelogCommandStep.java:143)
	at liquibase.Scope.lambda$child$0(Scope.java:216)
	at liquibase.Scope.child(Scope.java:225)
	at liquibase.Scope.child(Scope.java:215)
	at liquibase.Scope.child(Scope.java:194)
	at liquibase.Scope.child(Scope.java:282)
	at liquibase.Scope.child(Scope.java:286)
	at liquibase.command.core.helpers.DatabaseChangelogCommandStep.getDatabaseChangeLog(DatabaseChangelogCommandStep.java:138)
	at liquibase.command.core.helpers.DatabaseChangelogCommandStep.run(DatabaseChangelogCommandStep.java:84)
	at liquibase.command.CommandScope.lambda$execute$6(CommandScope.java:263)
	... 141 more
```

I think that output is getting suppressed for most of the tests (due to it happening a lot).

Not sure what to do with that though...

--- a/optimize/backend/pom.xml
+++ b/optimize/backend/pom.xml
@@ -724,6 +724,12 @@
       <groupId>io.camunda</groupId>
       <artifactId>configuration</artifactId>
       <scope>test</scope>
+      <exclusions>
+        <exclusion>
+          <groupId>org.liquibase</groupId>
+          <artifactId>liquibase-core</artifactId>
+        </exclusion>
+      </exclusions>
     </dependency>
     <dependency>
       <groupId>io.camunda</groupId>

--- a/optimize/backend/pom.xml
+++ b/optimize/backend/pom.xml
@@ -319,7 +319,6 @@
     <dependency>
       <groupId>com.github.wnameless.json</groupId>
       <artifactId>json-flattener</artifactId>
-      <version>0.18.0</version>
     </dependency>
 
     <dependency>

--- a/parent/pom.xml
+++ b/parent/pom.xml
@@ -2201,7 +2201,13 @@
       <dependency>
         <groupId>com.github.wnameless.json</groupId>
         <artifactId>json-base</artifactId>
-        <version>2.5.1</version>
+        <version>3.0.0</version>
+      </dependency>
+
+      <dependency>
+        <groupId>com.github.wnameless.json</groupId>
+        <artifactId>json-flattener</artifactId>
+        <version>0.18.0</version>
       </dependency>
     </dependencies>
   </dependencyManagement>

--- a/parent/pom.xml
+++ b/parent/pom.xml
@@ -67,6 +67,8 @@
     <version.identity>8.9.0-alpha2</version.identity>
     <version.instancio>5.5.1</version.instancio>
     <version.jackson>2.19.2</version.jackson>
+    <version.json-base>3.0.0</version.json-base>
+    <version.json-flattener>0.18.0</version.json-flattener>
     <version.junit>5.13.4</version.junit>
     <version.junit4>4.13.2</version.junit4>
     <version.log4j>2.25.1</version.log4j>
@@ -2201,13 +2203,13 @@
       <dependency>
         <groupId>com.github.wnameless.json</groupId>
         <artifactId>json-base</artifactId>
-        <version>3.0.0</version>
+        <version>${version.json-base}</version>
       </dependency>
 
       <dependency>
         <groupId>com.github.wnameless.json</groupId>
         <artifactId>json-flattener</artifactId>
-        <version>0.18.0</version>
+        <version>${version.json-flattener}</version>
       </dependency>
     </dependencies>
   </dependencyManagement>

